### ISSUE TITLE
The structure of dataset

### ISSUE THREAD

**Issue by GhTara** [Project-MONAI/research-contributions#60]
Hello,

First of all, thank you so much for this brilliant model. I have some questions.

1. I have downloaded the dataset from the link embedded in the notebook, but that dataset I have downloaded has just one label. Is it right, or have I downloaded it by mistake? 

2. Another question is about the structure of ground truth. My dataset has a binary map for each class. So I must merge these binary maps to become these labels into one channel aiming to use your model?

I appreciate any guidance.

**Comment by ahatamiz** [comment]
Hi @GhTara 

Would you please provide more information regarding this dataset ? is this the BTCV ? 

BTCV ground truth contains semantic labels for 13 different organs.

Best

**Comment by GhTara** [comment]
Yes, of course. According to the tutorial notebook, I have downloaded the dataset from this [link](https://www.synapse.org/#!Synapse:syn3193805/wiki/89480%5Cn).
I aim to process another dataset (my private dataset) to feed to UNETR. And my question is, how are the structures of ground truth. In my dataset, each organ is a NIFTI file as the binary map. I realized that I must be embedded all labels into one volume as ground truth instead of having multiple channels for each of them, (1, 128, 128, 128) instead of (number_of_organs, 128, 128, 128). 
I hope I understood correctly.

Many Thanks
